### ISSUE TITLE
games files.

### ISSUE THREAD

**Issue by Kakoryus** [zeropolis79/PETSCIIRobots-MSDOS#1]
Hello! 

I like to know if the games files will be available. They are not in the directory.

Thankyou

**Comment by zeropolis79** [comment]
No, I won't be including the game files. The directory contains the source code and source files, no executable code is included.
